Dockerfile.alpine

```diff
@@ -1,19 +1,19 @@
-FROM alpine:3.9
+FROM alpine:3.10
 
 LABEL \
 maintainer="ivan@better.com"
 
-ENV USER=app GOPATH=/go LC_ALL=en_US.utf-8
+ENV USER_ID=1000 USER=app GOPATH=/go LC_ALL=en_US.utf-8
```

Comment:
Whats this USER_ID? why 1000 for its value?

Reply:
fairly typical practice in Alpine land to create a non-root user. here's an instance in alpine-node: https://github.com/mhart/alpine-node/issues/48#issuecomment-370171836

the USER_ID env var itself may be used later to set the user ID numerically rather than by name, which can help with specific Kubernetes security policies

Reply:
Make sense. Thanks for the link 👍